stream_chat/client.py

```diff
@@ -156,15 +156,17 @@ def unflag_user(self, target_id, **options):
         data.update(options)
         return self.post("moderation/unflag", data=data)
 
-    def mute_user(self, target_id, user_id):
+    def mute_user(self, target_id, user_id, **options):
         """
         Create a mute
 
         :param target_id: the user getting muted
         :param user_id: the user muting the target
+        :param options: additional mute options
         :return:
         """
         data = dict(target_id=target_id, user_id=user_id)
+        data.update(options)
```

Comment:
`data = dict(target_id=target_id, user_id=user_id, **options)` might be more concise

Reply:
agreed, changed it. thanks.